vyper/parser/expr.py

```diff
@@ -262,7 +262,7 @@ def variables(self):
             return LLLnode.from_list(
                 var.pos,
                 typ=var.typ,
-                location='memory',
+                location=var.location,
```

Comment:
same here, could add a note that the `storage` location is precluded by the previous branch.

Reply:
:+1: